example/lib/pages/settings/settings-page.dart

```diff
@@ -0,0 +1,299 @@
+import 'package:example/pages/settings/settings-page-state.dart';
+import 'package:floater/floater.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class SettingsPage extends StatefulWidgetBase<SettingsPageState> {
+  SettingsPage() : super(() => SettingsPageState());
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _settingsAppBar(
+        goBack: this.state.goBack,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(20.0),
+        child: SingleChildScrollView(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              _displayInfo(),
+              _themeInfo(
+                themeNotifier: this.state.themeNotifier,
+                isDarkMode: this.state.isDarkMode,
+                toggleTheme: this.state.toggleTheme,
+              ),
+              _aboutInfo(
+                userName: this.state.userName,
+              ),
+              _statusInfo(),
+              _privacyPolicy(),
+              _logout(
+                logout: this.state.logout,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _displayInfo extends StatelessWidget {
+  const _displayInfo({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        SizedBox(
+          height: 20,
+        ),
+        const Row(
```

Comment:
why are you wrapping all this in row. there are multiple unnecessary rows

Reply:
i used it to get that in left side, i thought that is a simple way i can achieve that.

Reply:
it should be on left side. or if column is making it centered look at column documentation